### ISSUE TITLE
Aesthetic: Adjust Alignment of Secondary Nav Items

### ISSUE THREAD

**Issue by chadturnbull** [northlandchurch/Website-Styles#3]
I think the secondary nav alignment needs to be adjusted. It looks too visually pushed, like a piece of paper where someone writes through the side margins. My suggestion for the right set of words would be to make the "T" of "About" the right edge, and push everything left so that the "e" of "Tithe" is at the "T" of "About". 

The social media links may be fine where they are, but it would be worth looking at aligning the Twitter bird with the left edge of the "N" of "Northland".

**Comment by jhnwlsn** [comment]
@chadturnbull: I brought in the social and secondary navigations more towards the center of the page. Lining up the "Tithe" and "About" words made it a bit too asymmetrical, so I did a compromise and brought them both in the same distance. Take a look when you get a chance: http://northlandchurch.github.io/Website-Styles/. If you still feel indifferent about it, I'll most likely revert it to what it was (unless you want to see your proposed version) as the first version is better suited for future changes.

**Comment by chadturnbull** [comment]
I like what you have better than what was before. 

The sub nav is still hard to read and harder to read on the rollover. What if we put the sub nav directly below the main nav in the black bar (more visual contrast?) and then when the nav bar anchors it could even just get hidden except on rollover.
![cdc0a3af-cbbe-4707-90bc-6736b2bb36a5](https://cloud.githubusercontent.com/assets/10187666/8728011/e2bdd5da-2bb0-11e5-9960-0002198713c4.png)

**Comment by wondermade** [comment]
Seconds to that - I really like the weights and style on the navigation type save for the many moments when the sub nav is floating over another design elements. That would fix that problem and help preserve both the design and the necessary functionality.

**Comment by jhnwlsn** [comment]
I completely agree. I never felt that the nav was 100% there, so I just spent some time wrestling with it, and here's what I'm proposing:
![screen shot 2015-07-16 at 3 24 27 pm](https://cloud.githubusercontent.com/assets/2472911/8733030/5a10ae64-2bcf-11e5-893a-396a20976677.png)

Basically fixing the text over image issue. We could even do something cool with the ministries "fold-out" where it splits the blue and black rows of color and the content will come in there.

Let me know what y'all think! @chadturnbull @wondermade

**Comment by jhnwlsn** [comment]
This approach will also allow for more flexibility with future changes by allowing text to be over a solid color instead of images.

**Comment by wondermade** [comment]
What if they were flipped? (Northland on top, social icons on the bottom.)

**Comment by jhnwlsn** [comment]
What do y'all think? I'm fine with it on the bottom, it will just take some refactoring time to change some stuff that's already built.
![nav-02](https://cloud.githubusercontent.com/assets/2472911/8733843/fd0153d6-2bd3-11e5-8e5a-b62b33df04b3.jpg)

**Comment by chadturnbull** [comment]
We prefer the blue strip on the top. 

@wondermade Were you thinking the strip would move to the bottom, or were you wanting to flip-flop "Northland" and the social media links? (so that Northland is in the blue and the social media links are in the black)

**Comment by chadturnbull** [comment]
The blue stripe fixes the whole unbalanced thing for me as well.

**Comment by chadturnbull** [comment]
I think this is getting to be a much stronger navigation element. Good work on this John! 

A functionality thing that I think needs to happen is that it needs to anchor to the top of the browser, either from the beginning, (which would fix Nathan's concern about social media icons over Northland), or as you scroll down the page. Leaving the little space as you scroll makes the navigation feel invasive and ungrounded (both of which I interpret as negative things).

**Comment by chadturnbull** [comment]
The floating nature of this continues to feel incorrect. I don't mind the float upon first arrival because it really establishes the hero graphic and the flexibility of the navigation, but It starts to feel invasive as you move through the site. I'm also wondering if the black just makes it too heavy. The site really fights feeling too dark, so maybe a different color (grey, white, navy blue, or something) might brighten it up a bit.

**Comment by jhnwlsn** [comment]
I hear ya... The more I use the current navigation while working on this, the less I like it. It's a little too bespoke to feel like a solid navigation. Let's revisit the entire navigation later this week or next when we get into the design review phase.

**Comment by jhnwlsn** [comment]
Just noting here that #6 will be wrapped into this issue.
